qlib/data/dataset/handler.py

```diff
@@ -482,9 +494,7 @@ def process_data(self, with_fit: bool = False):
         Notation: (data)  [processor]
 
         # data processing flow of self.process_type == DataHandlerLP.PTYPE_I
-        (self._data)-[shared_processors]-(_shared_df)-[learn_processors]-(_learn_df)
-                                               \
-                                                -[infer_processors]-(_infer_df)
+        (self._data)-[shared_processors]-(_shared_df)-[learn_processors]-(_learn_df)-[infer_processors]-(_infer_df)
```

Comment:
We want to describe a branch logic at the `(_shared_df)-` node.

Reply:
Oh, sorry, my bad. I thoud that backslash here for line break)